guides/source/form_helpers.md

```diff
@@ -64,184 +64,229 @@ To create this form you will use `form_with` and the form builder object it yiel
 
 #### Radio Buttons
 
-Radio buttons, while similar to checkboxes, are controls that specify a set of options in which they are mutually exclusive (i.e., the user can only pick one):
+Radio buttons are form controls that only allow the user to select one option at a time from the list of choices.
+
+For example, radio buttons for choosing your favorite ice cream flavor:
 
 ```erb
-<%= form.radio_button :age, "child" %>
-<%= form.label :age_child, "I am younger than 21" %>
-<%= form.radio_button :age, "adult" %>
-<%= form.label :age_adult, "I am over 21" %>
+<%= form.radio_button :flavor, "chocolate_chip" %>
+<%= form.label :flavor_chocolate_chip, "Chocolate Chip" %>
+<%= form.radio_button :flavor, "vanilla" %>
+<%= form.label :flavor_vanilla, "Vanilla" %>
+<%= form.radio_button :flavor, "hazelnut" %>
+<%= form.label :flavor_hazelnut, "Hazelnut" %>
 ```
 
-Output:
+The above will generate the following HTML:
 
 ```html
-<input type="radio" id="age_child" name="age" value="child" />
-<label for="age_child">I am younger than 21</label>
-<input type="radio" id="age_adult" name="age" value="adult" />
-<label for="age_adult">I am over 21</label>
+<input type="radio" value="chocolate_chip" name="flavor" id="flavor_chocolate_chip">
+<label for="flavor_chocolate_chip">Chocolate Chip</label>
+<input type="radio" value="vanilla" name="flavor" id="flavor_vanilla">
+<label for="flavor_vanilla">Vanilla</label>
+<input type="radio" value="hazelnut" name="flavor" id="flavor_hazelnut">
+<label for="flavor_hazelnut">Hazelnut</label>
 ```
 
-The second parameter to [`radio_button`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-radio_button) is the value of the input. Because these two radio buttons share the same name (`age`), the user will only be able to select one of them, and `params[:age]` will contain either `"child"` or `"adult"`.
+The second parameter to [`radio_button`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-radio_button) is the value of the input. Because these radio buttons share the same name (`flavor`), the user will only be able to select one of them, and `params[:flavor]` will contain either `"chocolate_chip"`, `"vanilla"`, or `hazelnut`.
 
-NOTE: Always use labels for checkbox and radio buttons. They associate text with a specific option and,
-by expanding the clickable region,
-make it easier for users to click the inputs.
+NOTE: Always use labels for checkbox and radio buttons. They associate text with
+a specific option using the `for` attribute and, by expanding the clickable
+region, make it easier for users to click the inputs.
 
 ### Other Helpers of Interest
 
-Other form controls worth mentioning are text areas, hidden fields, password fields, number fields, date and time fields, and many more:
+There are many other form controls for text, email, password, date, and time. Below examples show some more helpers and their generated HTML.
```

Comment:
The way this sentence is worded perhaps suggests that this list is exhaustive (which it isn't). Maybe something like this would be clearer:

```suggestion
There are many other form controls including text, email, password, date, and time. The below examples show some more helpers and their generated HTML.
```